packages/block-library/src/gallery/test/use-get-new-images.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks';
+
+/**
+ * Internal dependencies
+ */
+import useGetNewImages from '../use-get-new-images';
+
+describe( 'gallery block useGetNewImages hook', () => {
+	let hook;
+	beforeEach( () => {
+		hook = renderHook(
+			( { images, imageData } ) => useGetNewImages( images, imageData ),
+			{
+				initialProps: { images: [ { test: 1 } ], imageData: [] },
+			}
+		);
+	} );
+
+	it( 'returns null if no images currently in the gallery', () => {
+		expect( hook.result.current ).toEqual( null );
+	} );
+
+	it( 'returns an empty array if the only images are those loaded from post content', () => {
```

Comment:
Just double checking that I'm understanding this test right from looking at the code for `useGetNewImages` — when we have `fromSavedContent` the goal is to hydrate the state inside `useGetNewImages` rather than treat them as "new" images. In that case, this test makes sense to me 👍 

I have a tiny suggestion for a tweak to the wording for this test, but it's totally optional — I just found writing out the test name helped me understand the expected behaviour 😀

```suggestion
	it( 'should not return images that are already loaded from saved post content', () => {
```